### PR TITLE
#162378667 Fix the dropdown for available rooms

### PR DESCRIPTION
**Pull request by ezrqnkemboi**
#### What does this PR do?

-Fix the drop-down for available rooms

#### Description of Task to be completed?

- Get rooms marked for maintenance during request of periods outside marked dates. 
- I should be able to see the room since it was marked as unavailable for a specific period in time that does not match the request period.

#### How should this be manually tested?

- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change directory - `cd travel_tool_back`
- Checkout to current branch - `git checkout ft-fix-dropdown-for-available-rooms-162378667`
- Run `make start`
- Mark a room to maintenance from date A-B. 
- Using `http://localhost:5000/api/v1/requests/` make a request which dates coincide with A-B. For example, departure being less than A and arrival is greater than B. 
- The easy way to test this app is to use frontend. Run the backend using this PR, then frontend from develop. 
- Mark room under maintenance at a certain period, and try the edge cases when making a new request. These rooms should not be available in time of maintenance but available outside time for maintenance. 

#### Any background context you want to provide?

- Need to be a manager, requester, Travel Admin

#### What are the relevant pivotal tracker stories?

[#162378667](https://www.pivotaltracker.com/story/show/162378667)

#### Screenshots (if appropriate)
screen shot 2018-12-07 at 11 15 54

#### Questions:

### CODE REVIEW

File: src/modules/guestHouse/__tests__/GuestHouseController.test.js [hunk 1]
```diff
@@ -210,18 +210,4 @@ describe('Guest Rooms - GET', () => {
         done();
       });
   });
-
```
Comment (samuelkubai) [inline src/modules/guestHouse/__tests__/GuestHouseController.test.js]: @ezrqnkemboi why did we delete this test?

**Comment by ezrqnkemboi** [comment]
@samuelkubai, Currently, one-way does not require getting beds.

**Comment by samuelkubai** [comment]
@ezrqnkemboi cool, got it. Nice work.
